### PR TITLE
README improvements

### PR DESCRIPTION
**Pull request by 0x9900**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/0x9900/genpass/pull/1" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ Generate Internet Password
 Generate a passworg base on the domain name of a website, your userid
 on this website and a key.
 
+This tool we generate the same password for a given use and domain
+name as long as you use the same key.
+
```
Comment (0x9900) [inline README.md:9]: what's that?

Comment (0x9900) [inline README.md:9]: LGTM

Comment (0x9900) [inline README.md:9]: !completed

**Comment by 0x9900** [comment]
LGTM

**Review by 0x9900 — COMMENTED** [review]
LGTM

**Pull request merged**